Cargo.toml

```diff
@@ -16,37 +16,41 @@ features = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = ["fs", "cookie-secure"]
+default = ["fs", "cookie-secure", "serde"]
 docs = ["unstable"]
 unstable = []
 hyperium_http = ["http"]
 async_std = ["fs"]
 cookies = ["cookie"]
 cookie-secure = ["cookies", "cookie/secure"]
 fs = ["async-std"]
+serde = ["serde_qs", "serde_crate", "serde_json", "serde_urlencoded"]
 
 [dependencies]
-# features: async_std
-async-std = { version = "1.6.0", optional = true }
+rand = "0.7.3"
```

Comment:
explanation for why unrelated liens changed: I moved the non-optional crates to the top of the list